### PR TITLE
K8s Generic Install

### PR DESCRIPTION
**Pull request by varunalla**
Updates the K8s Integration Installation document check.

### CODE REVIEW

**Review by nbrahms — APPROVED** [review]
Lgtm

File: src/plugins/kubeconfig/types.ts [hunk 1]
```diff
@@ -10,34 +10,21 @@ You should have received a copy of the GNU General Public License along with @p0
...
-export type EksClusterConfig = K8sClusterConfig & {
-  awsAccountId: string;
-  awsClusterArn: string;
-};
-
-type KubernetesProxyComponentConfig = {
-  isProxy: true;
+  isProxy: boolean;
+  token: string;
   publicJwk: string;
```
Comment (p0-andrewa) [inline src/plugins/kubeconfig/types.ts:19]: `publicJwk` is not required if `isProxy` is false; if we're not discriminating on `isProxy`, should this be optional?

**Review by p0-andrewa — COMMENTED** [review]
One small potential issue, otherwise LGTM

File: src/plugins/kubeconfig/types.ts (hunk 1 above)
Comment (varunalla) [reply src/plugins/kubeconfig/types.ts:19]: we could mark it as optional.

**Review by p0-andrewa — APPROVED** [review]

**Pull request merged**